.github/workflows/smoke-test-SlidesPANDAImageNetMILBenchmark.yml

```diff
@@ -0,0 +1,61 @@
+name: Smoke test SlidesPANDAImageNetMILBenchmark
+on:
+  push:
+    branches: [ main ]
+    tags:
+      - '*'
+  pull_request:
+    branches: [ main ]
+    paths:
+      - "hi-ml-cpath/**"
+      - ".github/workflows/cpath-pr.yml"
+      # When the core hi-ml libraries change, test the histo test too, to find problems with backwards compatibility
+      - "hi-ml-azure/**"
+      - "hi-ml/**"
+
+env:
+  pythonVersion: 3.7
+  folder: hi-ml-cpath
+  HIML_TENANT_ID: ${{ secrets.HIML_TENANT_ID }}
+  HIML_RESOURCE_GROUP: ${{ secrets.HIML_RESOURCE_GROUP }}
+  HIML_SUBSCRIPTION_ID: ${{ secrets.HIML_SUBSCRIPTION_ID }}
+  HIML_WORKSPACE_NAME: ${{ secrets.HIML_WORKSPACE_NAME }}
+  HIML_SERVICE_PRINCIPAL_ID: ${{ secrets.HIML_SERVICE_PRINCIPAL_ID }}
+  HIML_SERVICE_PRINCIPAL_PASSWORD: ${{ secrets.HIML_SERVICE_PRINCIPAL_PASSWORD }}
+
+jobs:
+
+  smoketest:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          lfs: true
+
+      - name: create config file
+        run: ./create_config.sh
+
+      - name: Set up Python ${{ env.pythonVersion }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.pythonVersion }}
+
+      - name: Install required packages
+        run: |
+          cd ${{ env.folder }}
+          make pip_from_conda
+
+      - name: Install hi-ml locally
+        run: |
+          cd hi-ml
+          make pip_local
+
+      - name: Install hi-ml-azure locally
+        run: |
+          cd hi-ml-azure
+          make pip_local
+
+      - name: smoke test
+        run: |
+          cd ${{ env.folder }}
+          make smoke_test_slidespandaimagenetmil_aml
```

Comment:
is there a reason why this is a separate workflow and not part of cpath-pr?

Reply:
I kept it separate as I had envisaged this becoming a file of all smoke tests that we want to run and it felt like a sensible separation but maybe it does make more sense to keep it under the Pathology PR heading - thoughts?

Reply:
I would suggest keeping it under the Pathology PR as the current checks are module specific. Also, we might also want to add some smoke tests to hi-ml-multimodal in the future... where should these live then?

Reply:
Makes sense to me. Have moved it